CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Fixed receiver address validation by disallowing rewards addresses ([PR 2781](https://github.com/input-output-hk/daedalus/pull/2781))
+- Fixed immediate language updates of application top menu bar ([PR 2813](https://github.com/input-output-hk/daedalus/pull/2813))
```

Comment:
Please put this one at the top of the list for fixes

Reply:
Done :+1: